com.unity.ml-agents/Editor/RenderTextureSensorComponentEditor.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+using UnityEditor;
+using MLAgents.Sensors;
+namespace MLAgents.Editor
+{
+    [CustomEditor(typeof(RenderTextureSensorComponent))]
+    [CanEditMultipleObjects]
+    internal class RenderTextureSensorComponentEditor : UnityEditor.Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            var so = serializedObject;
+            so.Update();
+
+            // Drawing the RenderTextureComponent
+            EditorGUI.BeginChangeCheck();
+
+            EditorGUI.BeginDisabledGroup(Application.isPlaying);
+            {
+                EditorGUILayout.PropertyField(so.FindProperty("m_RenderTexture"), true);
+                EditorGUILayout.PropertyField(so.FindProperty("m_SensorName"), true);
+                EditorGUILayout.PropertyField(so.FindProperty("m_Grayscale"), true);
+                EditorGUILayout.PropertyField(so.FindProperty("m_Compression"), true);
+            }
+
+            EditorGUI.EndDisabledGroup();
+
+            if (EditorGUI.EndChangeCheck())
```

Comment:
I do not think it is useful to check for changes is nothing happens when something changed

Reply:
Revised based on feedback from @awjuliani - the Camera and compression type can now be changed during runtime (since they don't affect the observation shape).